man/udevadm.xml

```diff
@@ -419,6 +419,12 @@
             same time.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>--ping</option></term>
+          <listitem>
+            <para>Send a ping message.</para>
```

Comment:
I think this needs to mention that it waits for the daemon to respond, with a configurable timeout.